src/Plutarch/Api/V1/ScriptContext.hs

```diff
@@ -216,3 +222,55 @@ pvalidatorHashFromAddress = phoistAcyclic $
         pmatch (pfromData $ pfield @"credential" # addr) $ \case
             PScriptCredential ((pfield @"_0" #) -> vh) -> pcon $ PJust vh
             _ -> pcon $ PNothing
+
+{- | Construct an address (without a staking credential) from a @PValidatorHash@
+
+    @since 1.1.0
+-}
+paddressFromValidatorHash ::
```

Comment:
Can we combine `paddressFromValidatorHash'` and `paddressFromValidatorHash` with something like `Term s (PValidatorHash :--> PMaybe PStakingCredential :--> PAddress)`?

Reply:
I'm open to this, but its the difference between potentially doing a `pcon . PDNothing $ (pdnil)` or not, which seems annoying. What do you think?

Reply:
It will be `pcon PNothing`. It would still look a little be ugly, but I think having one function is much cleaner.

Reply:
Nah, it's definitely `pcon . PDNothing $ pdnil`

But either way, I think its nbd. Fixed.

Reply:
that works, @chfanghr! I always forget that that's allowed...